blocks/init/src/Blocks/wrapper/wrapper-hooks.js

```diff
@@ -18,18 +22,70 @@ const parentComponentBlock = createHigherOrderComponent((BlockListBlock) => {
 				{},
 				innerProps,
 				{
-					className: globalManifest.globalVariables.customBlocksName,
+					className: `${globalManifest.globalVariables.customBlocksName} ${innerProps.attributes.blockClass}`,
 				}
 			);
 		}
 
-		return wp.element.createElement(
-			BlockListBlock,
-			updatedProps
-		);
+		return <BlockListBlock {...updatedProps} wrapperProps={{ 'data-id': clientId }} />;
 	};
 }, 'parentComponentBlock');
 
+const addListBlockClassName = (settings, name) => {
+	const blockNameSplit = name.split('/');
+	const blockNamespace = blockNameSplit[0];
+	const blockName = blockNameSplit[1];
+
+	if ((blockNamespace === namespace)) {
+		return settings;
+	}
+
+	const common = {
+		blockName: {
+			type: 'string',
+			default: blockName,
+		},
+		blockFullName: {
+			type: 'string',
+			default: name,
+		},
+		blockClass: {
+			type: 'string',
+			default: `wp-block-${blockName}`,
+		},
+		blockJsClass: {
+			type: 'string',
+			default: `js-wp-block-${blockName}`,
+		},
+	};
+
+	return assign( {}, settings, {
+		attributes: assign( {}, settings.attributes, wrapperManifest.attributes, common),
+	});
+};
+
+const withInspectorControls = createHigherOrderComponent( ( BlockEdit ) => {
+	return (props) => {
+		const {
+			name,
+		} = props;
+
+			return (
+					<>
+						{(name.split('/')[0] !== namespace) ?
+							<Wrapper props={props}>
+								<BlockEdit { ...props } />
+							</Wrapper> 
+							:
+							<BlockEdit { ...props } />
+						}
```

Comment:
```suggestion
						{name.split('/')[0] !== namespace
							? <Wrapper props={props}>
								<BlockEdit { ...props } />
							  </Wrapper> 
							: <BlockEdit { ...props } />
						}
```

Looks more readable to me, and I think we split these kinds of multiline statements in PHP the same way (the operator is on the same line as the following statement).